ql/src/test/org/apache/hadoop/hive/ql/txn/compactor/TestCleaner2.java

```diff
@@ -23,6 +23,6 @@
 public class TestCleaner2 extends TestCleaner {
   @Override
   boolean useHive130DeltaDirName() {
-    return false;
+    return true;
```

Comment:
Why is this change? Is this related to the PR?

Reply:
no, but we have TestCleaner & TestCleaner2 running the same stuff